src/content/getting-started/cloud-provider-accounts/cluster-api/azure/index.md

```diff
@@ -0,0 +1,138 @@
+
+First necessary step of configuring the Azure environment is to grant access for Giant Swarm Ops/Support to your subscription in order to provide 24/7 support. Access to the portal is important part of the provided support, where in some cases manual interventions have to take place.
+Easiest way is to use the Azure Lighthouse service, that allows to delegate the management of resources to third parties. While following this part of the guide, you will allow the Giant Swarm Staff group to manage your resources. This is beneficial as you will not have to manage access for each person separately within your subscription, but you will be adding a managed group that is kept up to date with the current active Giant Swarm Staff from our side.
+
+We require a built in role `Contributor` to access the resources that Giant Swarm is deploying and it can be used by default from the [Azure RBAC](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles).
+
+You can also create your own role assignment with restrictions to access specific Resource Groups, however it needs to be remembered to refresh the role with every newly created cluster so we can provide full support starting from the clusters creation.
+
+When all is set you can simply run this command:
+
+```nohighlight
+az deployment create --name <deploymentName (unique by subscription)> \
+                     --location <AzureRegion> \
+                     --template-file delegatedResourceManagement.json \
+                     --parameters delegatedResourceManagement.parameters.json \
+                     --verbose
+```
+
+You will have to supply a general Delegated Resource Management [template file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.json).
+
+The Delegated Resource Management template uses a [parameters file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.parameters.json) to supply the needed variables for configuration.
+Please remember to change the `roleDefinitionId` in case you would like to use your custom role definition. Moreover ask your Account Engineer so he can provide you the `GiantSwarmPrincipalID` and `GiantSwarmTenantID`
+
+## Accept legal terms for deployment of Flatcar image
+
+Giant Swarm deploys [Flatcar](https://www.flatcar-linux.org/) image developed by Kinvolk from Azure Marketplace. In order to be able to run the image, it is required by Azure to accept the legal terms.
+Please run the following command prior to creating a cluster on a given subscription:
+
+```nohighlight
+az vm image terms accept --offer flatcar-container-linux-free --plan stable --publisher kinvolk
+```
+
+This acceptance should be performed once for a subscription that is used to run Giant Swarm workload clusters.
+
+## Configure Azure subscription configuration for Management and Workload Clusters deployments
```

Comment:
```suggestion
### Configure Azure subscription configuration for Management and Workload Cluster deployments
```
In line 84 you said that previous points are prerequisites, so the need another level